.github/dependabot.yaml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    target-branch: "dependabot-updates"
```

Comment:
I believe `target-branch` is the branch to check for updates, not the branch to push the updates to.

Reply:
i.e. the default is `master`

Reply:
The way I understand the docs it's the branch which is looked at for the pip- or lockfile AND the target branch of the created PR: https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/configuration-options-for-dependency-updates#target-branch

Reply:
ok I see what you mean - lets try it